sklearn/metrics/pairwise.py

```diff
@@ -577,8 +579,12 @@ def _argmin_min_reduce(dist, start):
     return indices, values
 
 
+def _argmin_reduce(dist, start):
+    return dist.argmin(axis=1)
+
+
 def pairwise_distances_argmin_min(
-    X, Y, *, axis=1, metric="euclidean", metric_kwargs=None
+    X, Y, *, axis=1, metric="fast_euclidean", metric_kwargs=None
```

Comment:
The metric parameter of this docstring should be updated to document `"fast_euclidean"` and `"fast_sqeuclidean"`.